src/fu-util.c

```diff
@@ -1593,6 +1593,83 @@ fu_util_report_history_force(FuUtilPrivate *priv, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_report_export(FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
presumably `values` should accept GUID or device ID

Reply:
I guess it *could* but fall back to everything perhaps?

Reply:
Yup, just like other commands that accept GUID/device ID work.  Make sure to update bash completion too for this change.

Reply:
Fixed.